src/math/Vector2.js

```diff
@@ -388,10 +381,7 @@ Object.assign( Vector2.prototype, {
 		// computes the angle in radians with respect to the positive x-axis
 
 		var angle = Math.atan2( this.y, this.x );
-
-		if ( angle < 0 ) angle += 2 * Math.PI;
-
-		return angle;
+		return angle < 0 ? angle + 2 * Math.PI : angle;
```

Comment:
Maybe like so:
```js
return ( angle < 0 ) ? ( angle + 2 * Math.PI ) : angle;
```
It's a bit easer to read.

Reply:
OK.